vioscsi/helper.c

```diff
@@ -109,7 +109,7 @@ ENTER_FN_SRB();
         srbExt->out, srbExt->in,
```

Comment:
On virtqueue_add_buf_split(), can you fix up the comments above the function, so that you say returns VQ_ADD_BUFFER_SUCCESS on success?

Reply:
I could but that would be crossing over into [virtio], and then we would need to probably add `VQ_ADD_BUFFER_SUCCESS` into `virtio_ring.h` instead which on reading the head of the file might be a little fraught. I also don't see any maintainer in CODEOWNERS.